creator-node/scripts/verifyHealthCheckFileUpload.js

```diff
@@ -0,0 +1,73 @@
+const axios = require('axios')
```

Comment:
can you add a comment explaining what this file does? we should avoid merging scripts like this with zero documentation